src/handlers/process_tweets.py

```diff
@@ -42,9 +42,18 @@ def parse_message(msg: str, prefix: str) -> Tuple[str, str]:
 
 
 def validate_signature(pubkey: str, signed_message: str, twitter_handle: str):
-    for msg in [twitter_handle, f"@{twitter_handle}"]:
-        if is_sig_valid(sig=signed_message, msg=msg, pub_key=pubkey):
-            return
+    # Check different cases:
+    # - start with @ or without
+    # - end with whitespace
+    # - handle upper and lowercase
+    for whitechar in ["", " ", "\n", "\t"]:
```

Comment:
It's possible Windows user end up submitting `\r\n` since that's the line ending for that OS.

Reply:
Thinking more, perhaps you could use `.strip()` to get rid of most leading and trailing whitespace.

Reply:
this is the other way around:
- I am constructing different versions of a `msg` (with or without the whitespace),
- and checking if any of the version matches the signature.

Reply:
Oh yes :-|